modules/gdscript/gdscript_compiler.cpp

```diff
@@ -34,7 +34,7 @@
 
 bool GDScriptCompiler::_is_class_member_property(CodeGen &codegen, const StringName &p_name) {
 
-	if (!codegen.function_node || codegen.function_node->_static)
+	if (codegen.function_node && codegen.function_node->_static)
```

Comment:
Not exactly sure what the context of this method is, but this doesn't return `false` when `codegen.function_node` is `NULL` anymore while previously it did.

Not sure how important that is, if at all.